ISSUE_TEMPLATE/chore.md

```diff
@@ -0,0 +1,9 @@
+### Chore
+
+**Description**
+
+Please describe the task you want to update/upgrade.  
```

Comment:
`task you want to update/upgrade.`

We don't upgrade tasks

Reply:
chore is not always about upgrading

Reply:
PR updated